cli/src/main/resources/schema/wildfly-cli_3_3.xsd

```diff
@@ -0,0 +1,287 @@
+            <xs:element name="alias" type="xs:string" minOccurs="0" />
+            <xs:element name="key-password" type="xs:string" minOccurs="0" />
+            <xs:element name="trust-store" type="xs:string" minOccurs="0" />
+            <xs:element name="trust-store-password" type="xs:string" minOccurs="0" />
+            <xs:element name="modify-trust-store" type="xs:boolean" default="true" minOccurs="0">
+                <xs:annotation>
+                    <xs:documentation>
+                        Setting to true will cause the CLI to prompt when unrecognised certificates are received and allow
+                        them to be stored in the truststore.
+                    </xs:documentation>
+                </xs:annotation>
+            </xs:element>
+        </xs:sequence>
+    </xs:complexType>
+
+    <xs:complexType name="vaultType">
+        <xs:annotation>
+            <xs:documentation>
+                <![CDATA[
+                    Vault Configuration. If no 'code' and 'module' are supplied the default implementation will be used.
+                    If 'code' is specified but no 'module', it will look for the class named by 'specified' in the picketbox module.
+                    If 'module' is specified, it will look for the class specified by 'code' in the module specified by 'module'.
+                ]]>
+            </xs:documentation>
+        </xs:annotation>
+        <xs:sequence>
+            <xs:element name="vault-option" type="propertyType" minOccurs="0" maxOccurs="unbounded"/>
+        </xs:sequence>
+        <xs:attribute name="code" type="xs:string" use="optional"/>
+        <xs:attribute name="module" type="xs:string" use="optional"/>
+    </xs:complexType>
+
+    <xs:complexType name="propertyType">
+        <xs:attribute name="name" use="required"/>
+        <xs:attribute name="value" use="optional"/>
+    </xs:complexType>
+
+    <xs:complexType name="colorConfigType">
+        <xs:annotation>
+            <xs:documentation>
+                Enables and configures the CLI color output. Only foreground colors are configurable.
+                Available colors: black, red, green, yellow, blue, magenta, cyan, white and default.
+            </xs:documentation>
+        </xs:annotation>
+        <xs:sequence>
+            <xs:element name="enabled" type="xs:boolean" default="true" minOccurs="1" />
+            <xs:element name="error-color" type="xs:string" default="red" minOccurs="0" />
+            <xs:element name="warn-color" type="xs:string" default="yellow" minOccurs="0" />
+            <xs:element name="success-color" type="xs:string" default="default" minOccurs="0" />
+            <xs:element name="required-color" type="xs:string" default="magenta" minOccurs="0" />
+            <xs:element name="workflow-color" type="xs:string" default="green" minOccurs="0" />
+            <xs:element name="prompt-color" type="xs:string" default="blue" minOccurs="0" />
```

Comment:
It would be nice to have here enum type with all available colors, so wrong values would be caught by XML parser.

Reply:
What would be in the enum? The element or the colors?

Reply:
The colors.